src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -83,6 +84,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case SetCommand.COMMAND_WORD:
             return new SetCommandParser().parse(arguments);
 
+        case IngredientViewSingleCommand
+                .COMMAND_WORD:
```

Comment:
This can be placed in one line

Reply:
It is automatic fomatting.. can I leave it as that?

Reply:
noted with thanks

Reply:
I think it is better to keep it in one line, so that it is standardised with the other case statements.